app/modules/account.js

```diff
@@ -33,6 +41,8 @@ export default (state = {wif: null, address:null, loggedIn: false}, action) => {
       return {...state, wif:action.wif, address:loadAccount.address, loggedIn:true};
     case LOGOUT:
       return {'wif': null, address: null, 'loggedIn': false};
+    case SHOWKEY:
+      return {showKey: action.showKey};
```

Comment:
Do you also need to do `{...state, showKey: action.showKey};` to allow the rest of the state to propagate down into the component?